src/blog/2017-12-01/babel-support.pug

```diff
@@ -0,0 +1,49 @@
+extends ../../layout/blog-layout.pug
+block content
+    :markdown-it
+        Stryker now supports mutating and testing Babel projects!
+        The installation/upgrade scenario's in this blog post can be circumvented by using the [Quickstart](https://stryker-mutator.github.io/quickstart.html) if you don't already use Stryker in your project.
+    
+         ![](../../images/blogs/love-babel.png)
+
+        # New JavaScript mutator
+        In order to support mutating more recent JavaScript features as well as experimental features, we had to change the way we mutate JavaScript code. 
+        We had already released a separate plugin for mutating TypeScript code, but now we have also released a separate plugin for mutating JavaScript code. 
+        It is called the [stryker-javascript-mutator](https://www.npmjs.com/package/stryker-javascript-mutator) and it replaces the current 'ES5' mutator that has always come with Stryker. With this release, the 'ES5' has been deprecated and it will be removed in the future.
+
+        If you use Stryker with JavaScript code on your project, we highly suggest upgrading to the new stryker-javascript-mutator. You can do so by installing the package:
+        ```
+        npm i -D stryker-javascript-mutator
+        ```
+        And configuring it in your stryker.conf.js:
+        ```
+        mutator: 'javascript',
+        ```
+
+        If you have a plugins section in your config, please consider removing it or adding the styker-javascript-mutator to the plugins section if you really need the plugins section.
+
+        # Babel transpiler
+        Even if Stryker can mutate JavaScript code in which new features are used, your environment may not be able to run that code. 
+        Many people use Babel to convert their modern JavaScript to a format that will run environments that do not support these features.
+        We developed a new plugin for Stryker, the [stryker-babel-transpiler](https://www.npmjs.com/package/stryker-babel-transpiler), during a hackathon sponosored by [Info Support](https://opensource.infosupport.com). 
+        This plugin is capable of transpiling code writen using Babel to code that runs in your enviroment. Give Stryker your babel config and we'll make it work!
+        
+        To use the new Babel transpiler, start by installing the package:
+        ```
+        npm i -D stryker-babel-transpiler
```

Comment:
Probably better to write the full command here: `npm install --save-dev stryker-babel-transpiler`